src/test/functional/mutationServices.test/tagsMutationService-create.integration.ts

```diff
@@ -255,9 +265,9 @@ describe('Mutation for Tag: ', () => {
       variables,
     });
     expect(res.errors).to.undefined;
-    expect(res.data.createTags).to.deep.equal([
-      { name: 'kamille', savedItems: [{ id: '0' }] },
-    ]);
+    const data = res.data.createTags;
+    expect(data[0].name).equals('kamille');
```

Comment:
If you're going to change this from deep equal you should also assert length == 1